src/node.h

```diff
@@ -120,6 +120,50 @@ inline v8::Local<v8::Value> UVException(int errorno,
                      path);
 }
 
+typedef double async_id;
```

Comment:
Would it be possible to have this in src/node_api_backport.h and src/node_api_backport.cc as opposed to node.h and node.cc as I'm wondering if adding it to node.h would be semver minor.  The idea being to add it just for use in N-API as opposed to non N-API

Reply:
And to be clear I meant the whole of CallBackScope being in node_api_backport.xx